### DISCUSSION TITLE
How to switch of debug.log file generation?

### DISCUSSION THREAD

**Discussion by lofti198** [cefsharp/CefSharp#4071] [Q&A] [answered]
How to switch of debug.log file generation? Sometimes it appears to become too huge!)

**Comment by amaitland** [comment, marked as answer]
You can change the http://cefsharp.github.io/api/100.0.x/html/P_CefSharp_CefSettingsBase_LogSeverity.htm property to disable logging. 

Changing to log fatal errors only is recommend for debugging purposes.

**Reply by lofti198** [reply]
Thank you! Resolved!
